languageSpec/spec.md

```diff
@@ -913,14 +913,62 @@ need not themselves be manifest.
 If it is necessary for the current object to access an overridden attribute
 of a parent, the overridden
 attribute can be given an additional name by attaching
-an **`alias`** clause to the inherit or use statement: `alias n(_) = m(_)`
-creates a new `confidential` _alias_ `n(_)` for the attribute `m(_)`.
+an **`alias`** clause to the inherit or use statement: `alias new(_) = old(_)`
+creates a new `confidential` _alias_ `new(_)` for the attribute `old(_)`.
 It is a _object composition error_ to alias an attribute to its own name.
 Attributes of the parent that are not wanted can be excluded using
 an **`exclude`** clause: excluded attributes are replaced by a
 `confidential`, `required` method.
```

Comment:
`required` methods don't really exist— they are just markers.  So it makes no sense to say that they are `confidential`.  _Any_ attempt to execute such a method will fail.